Gems/Multiplayer/Code/Include/Multiplayer/MultiplayerTypes.h

```diff
@@ -156,6 +158,18 @@ namespace Multiplayer
         return serializer.IsValid();
     }
 
+    inline void PrefabEntityId::Reflect(AZ::ReflectContext* context)
+    {
+        if (AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
+        {
+            serializeContext->Class<PrefabEntityId>()
+                ->Version(1)
+                ->Field("prefabName", &PrefabEntityId::m_prefabName)
```

Comment:
The style for the field name is meant to be more English-like and less like a C++ parameter name. So "Prefab Name" instead of "prefabName".